flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -915,6 +916,16 @@ public TableResultInternal executeInternal(Operation operation) {
             return executeInternal(((StatementSetOperation) operation).getOperations());
         } else if (operation instanceof ExplainOperation) {
             ExplainOperation explainOperation = (ExplainOperation) operation;
+            if (explainOperation instanceof ExplainFileOperation) {
+                ExplainFileOperation explainfileOperation = (ExplainFileOperation) explainOperation;
+                CompiledPlan cp =
+                        loadPlan(PlanReference.fromFile(explainfileOperation.getFilePath()));
```

Comment:
Why just load the plan and return it?  I think we should call `compiledPlan#explain(...)`

Reply:
I got it and fix it with `compiledPlan#explain`

Reply:
Ok. I fixed it.